src/ui/public/index_patterns/_get_ids.js

```diff
@@ -16,7 +16,7 @@ export default function GetIndexPatternIdsFn(es, kbnIndex) {
     cachedPromise = es.search({
       index: kbnIndex,
       type: 'index-pattern',
-      fields: [],
+      storedFields: [],
```

Comment:
@cjcenizal and I took a look at this, kibana only needs the ids so this is asking for less data.